lib/pages/chat/chat.dart

```diff
@@ -645,10 +645,19 @@ class ChatController extends State<Chat>
     }
   }
 
-  void copySingleEventAction() async {
-    if (selectedEvents.length == 1) {
+  void copySingleEventAction({Event? event}) async {
+    if (event != null) {
+      return event.copy(context, timeline!);
+    }
+
+    if (selectedEvents.isNotEmpty) {
       await selectedEvents.first.copy(context, timeline!);
+      return;
     }
+
+    Logs().e(
+      'copySingleEventAction():: event is null and selectedEvents is empty',
+    );
```

Comment:
How about you keed this method unchanged

Reply:
At the moment we can't copy something without selecting it because we forced the selection before copying. That's why I added a way to copy one event individually